### ISSUE TITLE
Accessing your Contentful space failed.

### ISSUE THREAD

**Issue by brentarias** [rohitguptab/rg-portfolio#1]
This is my first time trying to use your Gatsby starter.  When I imported the media to contentful from `content-data.json`, I got the following error messages:

```
Finished importing all data
The import took a minute (88s)

The following 4 errors and 11 warnings occurred:

23:04:23 - Rate limit error occurred. Waiting for 1506 ms before retrying...
23:04:23 - Rate limit error occurred. Waiting for 1660 ms before retrying...
23:04:23 - Rate limit error occurred. Waiting for 1623 ms before retrying...
23:04:36 - Rate limit error occurred. Waiting for 1652 ms before retrying...
23:04:37 - Rate limit error occurred. Waiting for 1505 ms before retrying...
23:04:95 - Rate limit error occurred. Waiting for 1664 ms before retrying...
23:04:10 - Rate limit error occurred. Waiting for 1639 ms before retrying...
23:04:63 - AssetProcessingTimeout: Asset is taking longer then expected to process.
23:04:85 - AssetProcessingTimeout: Asset is taking longer then expected to process.
23:04:15 - AssetProcessingTimeout: Asset is taking longer then expected to process.
23:04:38 - AssetProcessingTimeout: Asset is taking longer then expected to process.
23:05:10 - Rate limit error occurred. Waiting for 1648 ms before retrying...
23:05:36 - Rate limit error occurred. Waiting for 1542 ms before retrying...
23:05:39 - Rate limit error occurred. Waiting for 1554 ms before retrying...
```

I ignored the errors and then configured my space ID and access token within `gatsby-config.js`.  When I ran `gatsby devleop` I got this error:

```
PS C:\repos\gatsby\artisan> gatsby develop
success open and validate gatsby-configs - 0.223s
success load plugins - 23.921s
success onPreInit - 0.021s
success initialize cache - 0.100s
success copy gatsby files - 1.821s
success onPreBootstrap - 0.193s
success createSchemaCustomization - 0.021s
Starting to fetch data from Contentful
info Fetching default locale

 ERROR

Accessing your Contentful space failed.
Try setting GATSBY_CONTENTFUL_OFFLINE=true to see if we can serve from cache.

Used options:
spaceId: "*********ctr"
accessToken: "*********************************************MBWk"
host (default value): "cdn.contentful.com"
environment (default value): "master"
downloadLocal (default value): false
localeFilter (default value): [Function]
forceFullSync (default value): false
pageLimit (default value): 100
useNameForId (default value): true

not finished source and transform nodes - 0.682s
```

I tried re-running the content import.  The second time around it had only warnings, no errors.  I went back and did another `gatsby develop`, but the result was the same error.

Do you have any suggestions?

**Comment by rohitguptab** [comment]
It looks like something wrong happens in Contentful data.

Let me check from my side I will get back to you.

Thanks

**Comment by brentarias** [comment]
I should have included in my first post my `gatsby --version` info:

```
C:\repos\gatsby\artisan>gatsby --version
Gatsby CLI version: 2.11.22
Gatsby version: 2.20.36
```

I think that is saying that the version used to create your starter is more recent than the version I have installed.  More version info:

 * node.js version is 12.13.0.
 * Windows 10 version 1903

**Comment by brentarias** [comment]
Take a look at this [seemingly identical issue on Stack Overflow](https://stackoverflow.com/questions/60892938/accessing-your-contentful-space-failed-with-gatsby-source-contentful).

**Comment by brentarias** [comment]
Looks like I resolved the problem.  The overly simple answer is that I suspected there was a discrepency between the Contentful models expected by the `rg-portfolio` starter, and what model(s) actually existed in my Contentful space.  I solved the problem by deleting the Contentful space, re-creating it, and re-running your script to automate the creation of models. 
I answered that Stack Overflow post with more details of how I reached the conclusion that about this solution approach.
